Lib/inspect.py

```diff
@@ -389,7 +389,7 @@ def classify_class_attrs(cls):
 
     mro = getmro(cls)
     metamro = getmro(type(cls)) # for attributes stored in the metaclass
-    metamro = tuple(cls for cls in metamro if cls not in (type, object))
+    metamro = tuple([cls for cls in metamro if cls not in (type, object)])
```

Comment:
The list comprehension is not necessary and doesn't improve the code.  This should stay as-is.